tests/test_code_report.py

```diff
@@ -1,48 +1,83 @@
-#!/usr/bin/env python
-# -*- coding: UTF-8 -*-
+#!/usr/bin/env python3
 
-from __future__ import absolute_import
+from typing import List
 import re
 import pytest
 
 
-@pytest.mark.nonci_applicable
-def test_code_report(universum_runner): #TODO: parametrize test for different versions of python
+@pytest.fixture(name='runner_with_pylint')
+def fixture_runner_with_pylint(universum_runner):
+    universum_runner.environment.install_python_module("pylint")
+    yield universum_runner
+
+
+def get_config(args: List[str]):
     config = """
 from _universum.configuration_support import Variations
 
-configs = Variations([dict(name="Run static pylint", code_report=True,
-                           command=["universum_pylint", "--python-version=3", "--files", "source_file.py",
-                           "--result-file", "${CODE_REPORT_FILE}"])])
-"""
+configs = Variations([dict(name="Run static pylint", code_report=True, command=['universum_pylint'"""
+    for arg in args:
+        config += f", '{arg}'"
+    return f"{config}])])"
```

Comment:
Why is this needed? It's only called once with one set of arguments, isn't it?

Reply:
not once